HeliPort/Appearance/StatusMenu.swift

```diff
@@ -100,9 +109,9 @@ final class StatusMenu: NSMenu, NSMenuDelegate {
             }
 
             // Create Network... has not been implemented in itlwm
-            items[items.count - 7].isHidden = true
+            items[items.count - 8].isHidden = true
 
-            for idx in 1...2 {
+            for idx in 1...4 {
```

Comment:
We really need some docs to tell people we're hiding those features in the option menu LOL